lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -298,21 +298,12 @@ def _monitor(self, poll_interval):
                     self._report_usage({})
             hub.sleep(poll_interval)
 
-    def _poll_stats(self, datapath):
+    def _poll_stats(self, datapath, cookie = 0, cookie_mask = 0):
         """
         Send a FlowStatsRequest message to the datapath
         """
-        ofproto, parser = datapath.ofproto, datapath.ofproto_parser
-        req = parser.OFPFlowStatsRequest(
-            datapath,
-            table_id=self.tbl_num,
-            out_group=ofproto.OFPG_ANY,
-            out_port=ofproto.OFPP_ANY,
-        )
-        try:
-            messages.send_msg(datapath, req)
-        except MagmaOFError as e:
-            self.logger.warning("Couldn't poll datapath stats: %s", e)
+        #call send stats request helper in flows.py
```

Comment:
rm comment/it doesn't add much value as we can already understand whats happening from the code